amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/rm/rest/api/impl/ApiNodesModelFactory.java

```diff
@@ -992,9 +1017,9 @@ private void mapPeriodProperties(DispositionActionDefinition dispositionActionDe
                 {
                     retentionScheduleActionDefinition.setPeriodAmount(Integer.parseInt(periodArray[1]));
                 }
-                catch (NumberFormatException e)
+                catch (Exception exception)
                 {
-                    throw new NumberFormatException("Error parsing period amount: " + e.getMessage());
+                    throw new NumberFormatException("Error parsing period amount: " + exception);
```

Comment:
Why have we changed catching NumberFormatException to Excaption ?

Reply:
done changes as expected.